neo/tools/compilers/dmap/dmap.cpp

```diff
@@ -364,7 +364,7 @@ void Dmap( const idCmdArgs& args )
 	// if this isn't a regioned map, delete the last saved region map
 	if( passedName.Right( 4 ) != ".reg" )
 	{
-		std::sprintf( path, "%s.reg", dmapGlobals.mapFileBase );
+		common->Printf( ( path, "%s.reg", dmapGlobals.mapFileBase ) );
```

Comment:
the original call formats the path char array, I think that common->Printf() just prints to the console.